audit-log-api/src/use-cases/validateCreateAuditLog.ts

```diff
@@ -42,12 +42,19 @@ export default (auditLog: AuditLog): ValidationResult => {
     errors.push("Received date must be ISO format")
   }
 
+  if (!createdBy) {
+    errors.push("Created by is mandatory")
+  } else if (typeof createdBy !== "string") {
+    errors.push("Created by must be string")
```

Comment:
I agree it would make it easier to maintain error messages across the code when we have so many of them. I would say we can review this later (when we have less urgent work) and refactor the code and move all error messages to a separate file or module.